src/board/system76/common/lid.c

```diff
@@ -26,8 +26,6 @@ void lid_event(void) {
                 pmc_swi();
                 lid_wake = false;
             }
-
-            kbled_enable(true);
```

Comment:
Why removing?

Reply:
It's enabled in `power.c` if the lid is open anyway. Removing this prevents the backlight flickering on resume.